runtime/src/bank.rs

```diff
@@ -3476,6 +3477,10 @@ impl Bank {
         timings.saturating_add_in_place(ExecuteTimingType::CheckUs, check_us);
 
         let (blockhash, lamports_per_signature) = self.last_blockhash_and_lamports_per_signature();
+        // TODO: Pass into `TransactionProcessingEnvironment` in place of
+        // `rent_collector` when SVM supports the new `SVMRentCollector` trait.
+        let _rent_collector_with_metrics =
+            RentCollectorWithMetrics::new(self.rent_collector.clone());
```

Comment:
We could elide this line until it becomes necessary.

Reply:
Yeah, I also could just not add it at all in this PR, but I wanted to demonstrate how the new type would be used. I can remove if you prefer.

Reply:
Maybe leaving the example in a comment still works.

Reply:
Is the underscore prefix insufficient for some reason? Next PR will make use of it.